week-1.py

```diff
@@ -31,35 +31,41 @@ class Player:
     
     # create here two local variables to store a unique ID for each player and the player's current 'pot' of money
     # [FILL IN YOUR VARIABLES HERE]
-    
+    playerID = 0
+    playerPot = 0
     # in the __init__() function, use the two input variables to initialize the ID and starting pot of each player
     
     def __init__(self, inputID, startingPot):
         # [CREATE YOUR INITIALIZATIONS HERE]
-        
+        self.playerID = inputID
+        self.playerPot = startingPot
     # create a function for playing the game. This function should take on input for the card of the dealer.
     # it should then take a random card from 
     
     def play(self, dealerCard):
         # [CREATE CODE FOR SELECTING A RANDOM CARD]
-        
+        self.dealrcard = random.choice(cards)
+        self.playerCard = random.choice(cards)
```

Comment:
If you are referencing these as local variables, you need to declare them within the object definition.